### PR TITLE
(refactor): Fixing the chaos engine snippet (permissions)

### PR DESCRIPTION
**Pull request by uditgaurav**
Signed-off-by: Udit Gaurav uditgaurav@gmail.com

-  Modify the dummy app PVC name in the chaosengine spec of openebs experiments.

### CODE REVIEW

**Review by ispeakc0de — DISMISSED** [review]
/lgtm

File: docs/openebs-target-container-failure.md [hunk 1]
```diff
@@ -228,7 +228,7 @@ spec:
             - name: TARGET_CONTAINER
               value: 'cstor-istgt'
             - name: APP_PVC
-              value: 'pvc-c466262a-a5f2-4f0f-b594-5daddfc2e29d'    
+              value: 'demo-nginx-claim'
```
Comment (chandankumar4) [inline docs/openebs-target-container-failure.md]: In appInfo percona is specfied but here nginx is there is's a confusing!!

Comment (chandankumar4) [reply docs/openebs-target-container-failure.md]: Check for other experiment also

**Comment by ksatchit** [comment]
Refer https://github.com/litmuschaos/litmus-docs/pull/200/

**Review by ispeakc0de — APPROVED** [review]
/lgtm

**Review by ksatchit — APPROVED** [review]

**Pull request merged**